packages/costa/src/index.ts

```diff
@@ -18,19 +15,20 @@ import * as dataCook from '@pipcook/datacook';
 import * as path from 'path';
 import Debug from 'debug';
 import { importFrom } from './utils';
+
 const debug = Debug('costa.runnable');
 
 export * from './types';
 
-export type DefaultRuntime = Runtime<Sample<any>, DatasetMeta>;
+export type DefaultRuntime = Runtime<dataCook.Dataset.Types.Sample<any>, dataCook.Dataset.Types.DatasetMeta>;
```

Comment:
dataCook or Datacook?

Reply:
@WenheLI What's your opinion？

Reply:
See https://github.com/alibaba/pipcook/pull/739/files#diff-731eeb09ecd79fd1c3157ce640aa046c8c798d2fdf133837b696e4e1bfa8c0c4R1

@FeelyChau we need a consistent name for the same package.

Reply:
Should be `Datacook`.